sites/fast-website/src/app/data/feature.data.ts

```diff
@@ -1,65 +1,133 @@
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Technology",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Web Standards",
         body:
             "All FAST components follow WCAG 2.1, are W3C spec-compliant and use the W3C interaction models when available.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Web Standards",
+        links: [
+            {
+                anchorText: "WCAG",
+                url: "https://www.w3.org/WAI/standards-guidelines/wcag/",
+            },
+            {
+                anchorText: "Web Accessibility Initiative",
+                url: "https://www.w3.org/WAI/",
+            },
+        ],
     },
     {
+        header: "Animation",
         body:
             "Design sophisticated animation sequences with the animation library, an interface for the Web Animations API.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Animation",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Colors",
         body:
             "Create color palettes, extract colors from images, and handle other color operations using our color library.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Colors",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Layout / Grid",
         body:
             "Build layouts such as a 12 column grid for content or an application grid with resizable panels.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Layout / Grid",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Other",
         body:
             "Leverage a toolkit of general utilities such as keyboarding, Right-To-Left (RTL), number, and string manipulation.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Other",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
```

Comment:
Let's drop these two - the layouts/grid stuff is currently only available in React. Dropping "other" will even out the section and remove the confusion that "other" currently adds.

Reply:
Removed both